wfdb/io/annotation.py

```diff
@@ -1764,13 +1765,14 @@ def load_byte_pairs(record_name, extension, pn_dir):
         The input filestream converted to an Nx2 array of unsigned bytes.
 
     """
+    file_name = pathlib.Path(record_name).with_suffix('.' + extension)
     # local file
     if pn_dir is None:
-        with open(record_name + '.' + extension, 'rb') as f:
+        with open(file_name, 'rb') as f:
```

Comment:
paths offer a `.open` function, too, which might be more readable than using the global one.